src/components/Navbar/styles.scss

```diff
@@ -0,0 +1,63 @@
+@import "../../styles/variables.scss";
```

Comment:
create a main.scss file and import it to App. then you don't need to add the theme variables to every page.